components/Cart/cart.css

```diff
@@ -123,6 +124,7 @@ input:focus{
 
 .order{
     width: 100%;
+    padding: 1.5rem 1.8rem
```

Comment:
try to keep your paddings consistent. avoid being very specific. 
in this case -  even if you give `padding: 1.5rem`. it looks fine.

Reply:
Done. Thanks for the suggestion.